README.md

```diff
@@ -1,7 +1,7 @@
 Distributed Machine Learning Common Codebase
 ============================================
 
-[![Build Status](https://travis-ci.org/dmlc/dmlc-core.svg?branch=master)](https://travis-ci.org/dmlc/dmlc-core)
+[![Build Status](https://github.com/leezu/dmlc-core/workflows/continuous%20build/badge.svg)](https://github.com/dmlc/dmlc-core/actions)
```

Comment:
is the change on the badge link intended?

Reply:
In principle yes, but to `/dmlc/dmlc-core/workflows/continuous%20build/badge.svg` instead of  `/leezu/dmlc-core/workflows/continuous%20build/badge.svg`